src/Compiler/SyntaxTree/ParseHelpers.fs

```diff
@@ -1066,46 +1066,79 @@ let leadingKeywordIsAbstract =
     | _ -> false
 
 /// Unified helper for creating let/let!/use/use! expressions
-/// Creates either SynExpr.LetOrUse or SynExpr.LetOrUseBang based on isBang parameter
+/// Creates SynExpr.LetOrUse based on isBang parameter
 /// Handles all four cases: 'let', 'let!', 'use', and 'use!'
 let mkLetExpression
     (
         isBang: bool,
```

Comment:
Why is the SyntaxTree field called `isComputed` whereas the argument here is `isBang` ?
To me, `isBang` is more apparent - if `isComputed` means its always coming from a `let!` or `use!` syntax.

Reply:
Sure. lets use isBang